trackomatic.js

```diff
@@ -48,60 +77,64 @@ function Trackomatic(tracker, config) {
   // Viewport tracking
   var viewportSize = getViewportSize();
   var viewportRatio = (viewportSize.width / viewportSize.height).toPrecision(2);
-  var simpleviewportSize = roundXtoY(viewportSize.width, 100)
+  var simpleviewportSize = roundXtoY(viewportSize.width, 100);
+
   dataLayer.push({
     'fed-viewportwidth'  : viewportSize.width,
     'fed-viewportheight' : viewportSize.height,
     'fed-viewportratio'  : viewportRatio
   });
+
   tracker.send('event', 'FED Viewport Size', String(simpleviewportSize), viewportRatio, 0, { 'nonInteraction': 1 });
 
   // Resizing tracking
   var diffDimensions = function() {
     var newSize = getViewportSize();
-    var event   = {};
+    var event = {};
+
     if (newSize.height > viewportSize.height) {
-    event['fed-resize-height'] = 'taller';
-    tracker.send('event', 'FED Viewport Resize', 'FED Resize Height', 'taller', 0, { 'nonInteraction': 1 });
+      event['fed-resize-height'] = 'taller';
+      tracker.send('event', 'FED Viewport Resize', 'FED Resize Height', 'taller', 0, { 'nonInteraction': 1 });
     } else if (newSize.height < viewportSize.height) {
-    event['fed-resize-height'] = 'shorter';
-    tracker.send('event', 'FED Viewport Resize', 'FED Resize Height', 'shorter', 0, { 'nonInteraction': 1 });
+      event['fed-resize-height'] = 'shorter';
+      tracker.send('event', 'FED Viewport Resize', 'FED Resize Height', 'shorter', 0, { 'nonInteraction': 1 });
     }
+
     if (newSize.width > viewportSize.width) {
-    event['fed-resize-width'] = 'wider';
-    tracker.send('event', 'FED Viewport Resize', 'FED Resize Width', 'wider', 0, { 'nonInteraction': 1 });
+      event['fed-resize-width'] = 'wider';
+      tracker.send('event', 'FED Viewport Resize', 'FED Resize Width', 'wider', 0, { 'nonInteraction': 1 });
     } else if (newSize.width < viewportSize.width) {
-    event['fed-resize-width'] = 'narrower';
-    tracker.send('event', 'FED Viewport Resize', 'FED Resize Width', 'narrower', 0, { 'nonInteraction': 1 });
+      event['fed-resize-width'] = 'narrower';
+      tracker.send('event', 'FED Viewport Resize', 'FED Resize Width', 'narrower', 0, { 'nonInteraction': 1 });
     }
+
     viewportSize = newSize;
     dataLayer.push(event);
   }
 
   if (window.addEventListener) {
-    if (!/iPad|iPhone|iPod/.test(navigator.platform) && !/android/i.test(navigator.userAgent)) {
+    if (!CONSTANTS.REGEX.IDEVICE.test(navigator.platform) && !CONSTANTS.REGEX.ANDROID.test(navigator.userAgent)) {
       window.addEventListener('resize', debounce(diffDimensions, 1000));
     }
     window.addEventListener('orientationchange', debounce(diffDimensions, 1000));
   }
 
-
   // Input method tracking
-
   var firstInputRecorded = false;
-  var mouseEvent         = 'mousedown';
-  var nonViewKeys        = {
+  var mouseEvent = 'mousedown';
+  var nonViewKeys = {
```

Comment:
Yeah, maybe "nonInteractive" is a better term here. These are keys that we're not tracking as first input because they're used for scrolling the page.